assets/components/button/placeholderButton.jsx

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import SharedButton, { defaultProps, type SharedButtonPropTypes } from './_sharedButton';
+import './button.scss';
+
+// ----- Render ----- //
+
+const PlaceholderButton = ({ modifierClasses, ...props }: SharedButtonPropTypes) => (
```

Comment:
seems to be the same as `NonInteractiveButton` - is this just a rename in progress?